addons/nebula/templates/cmpd-graphd.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: nebula-graphd-{{ .Chart.Version }}
+  labels:
+    {{- include "nebula.labels" . | nindent 4 }}
+spec:
+  provider: Community
+  description: {{ .Chart.Description }}
+  serviceKind: nebula
+  serviceVersion: {{ .Chart.AppVersion }}
+  runtime:
+    securityContext:
+      fsGroup: 1001
+    containers:
+      - name: graphd
+        command:
+          - /bin/bash
+          - -c
+          - |
+            set -ex
+            trap : TERM INT
+            exec /usr/local/nebula/bin/nebula-graphd --flagfile=/usr/local/nebula/etc/nebula-graphd.conf --meta_server_addrs=$NEBULA_METAD_SVC:9559 --local_ip=${KB_POD_FQDN}{{ .Values.clusterDomain }} --daemonize=false
```

Comment:
NEBULA_METAD_SVC is a string of "FQDN:PORT,FQDN:PORT,..." a list of fqdn and port, concatenated with comma.

Reply:
fixed